l10n_ch_import_cresus/wizard/l10n_ch_import_cresus.py

```diff
@@ -0,0 +1,344 @@
+            delimiter = '\t'
+            src.write(content)
+            with tempfile.TemporaryFile() as decoded:
+                src.seek(0)
+                base64.decode(src, decoded)
+                decoded.seek(0)
+                return self._prepare_csv_data(decoded, delimiter)
+
+    def _prepare_csv_data(self, csv_file, delimiter=","):
+        """Parse a decoded CSV file and return head list and data list
+
+        :param csv_file: decoded CSV file
+        :param delimiter: CSV file delimiter char
+        :returns: (head [list of first row], data [list of list])
+
+        """
+        try:
+            data = csv.DictReader(csv_file, fieldnames=self.HEAD_CRESUS,
+                                  delimiter=delimiter)
+        except csv.Error as error:
+            raise orm.except_orm(
+                _('CSV file is malformed'),
+                _("Please choose the correct separator \n"
+                  "the error detail is : \n %s") % repr(error)
+            )
+        # Generator does not work with orm.BaseModel.load
+        values = [x for x in data if x]
+
+        return (values)
+
+    def _manage_load_results(self, result):
+        """Manage the BaseModel.load function output and store exception.
+
+        Will generate success/failure report and store it into report field.
+        Manage commit and rollback even if load method uses PostgreSQL
+        Savepoints.
+
+        :param imp_id: current importer id
+        :param result: BaseModel.load returns
+                       {ids: list(int)|False, messages: [Message]}
+        :param _do_commit: toggle commit management only used
+                           for testing purpose only
+        :returns: current importer id
+
+        """
+        # Import sucessful
+        state = msg = None
+        if not result['messages']:
+            msg = _("Lines imported")
+            state = 'done'
+        else:
+            msg = self.format_messages(result['messages'])
+            state = 'error'
+        self.write({'state': state})
+        self.MOVE_IDS = result['ids']
+        return (state, msg)
+    
+    @api.multi
+    def _standardise_data(self, data):
+        """ This function split one line of the CSV into multiple lines.
+        Cresus just write one line per move, 
+        
+
+        """
+        new_openerp_data = []
+        tax_obj = self.env['account.tax']
+        user_obj = self.env['res.users']
+        account_obj = self.env['account.account']
+        cp = user_obj.browse([self.env.uid]).company_id
+        company_partner = cp.partner_id.name
+        standard_dict = dict(izip_longest(self.HEAD_OPENERP, []))
+        previous_date = False
+        for line_cresus in data:
+            is_negative = False
+            current_date_french_format = datetime.strptime(line_cresus['date'],
+                                                           '%d.%m.%Y')
+            current_openerp_date = datetime.strftime(
```

Comment:
Instead of datetime you can use fields.Datetime.

https://www.odoo.com/documentation/8.0/reference/orm.html#openerp.fields.Datetime

This to avoid having to import DEFAULT_SERVER_DATE_FORMAT